src/View/Helper/PaginatorHelper.php

```diff
@@ -1239,7 +1239,14 @@ public function implementedEvents()
      */
     public function limitControl(array $limits = [], $default = null, array $options = [])
     {
-        $out = $this->Form->create(null, ['type' => 'get']);
+        $fopt = array();
+        $fopt['type'] = 'get';        
+        if (isset($options['form'])) {
+            $fopt = array_merge($fopt,$options['form']);
```

Comment:
Are those assoc arrays (string key based)? If so + would be the more correct merge operator.
Otherwise array_merge() is fine.